src/style.css

```diff
@@ -1,3 +1,76 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  line-height: 1;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 60px;
+  padding: 0 50px;
+
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.header__logo {
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  white-space: nowrap;
+  margin: 0;
+  padding: 0;
+  height: 100%;
+}
+
+.nav__item {
+  list-style: none;
+  margin-right: 20px;
+  position: relative;
+}
+
+.nav__item:last-child {
+  margin-right: 0;
+}
+
+.nav__link {
+  display: inline-block;
+  color: inherit;
+  text-decoration: none;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  cursor: pointer;
+  padding: 24px 0;
+}
+
+.nav__link:hover,
+.is-active {
+  color: #00acdc;
```

Comment:
Move repeatable colors to CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)